docs/installation.rst

```diff
@@ -0,0 +1,6 @@
+Installation
+============
+
+.. include::  readme
```

Comment:
This wouldn't render the installation text in the sdk-build though, I think, since it only uses a symlink to the docs folder. I could just make a copy of the readme to the docs folder, as I think is done in other packages.